### PR TITLE
AMBR-822 Upgrade to tomcat 8.5

### PR DESCRIPTION
**Pull request by egh**
*JIRA issue:* https://jira.plos.org/jira/browse/AMBR-822

*Related PRs:* 

- https://github.com/PLOS-Formulas/wombat-formula/pull/115

## What this PR does:

Upgrades wombat to use tomcat 8.5

## Developer Notes

Testing:

1. Checkout the `AMBR-822` branch in `wombat-formula`.
2. Checkout the `AMBR-822` branch in `wombat`
3. Run `mvn clean package` in `wombat`
4. Run `pogos vigor vagrant up apps/ambra`
5. Follow steps in https://github.com/PLOS/molten/blob/dev/vagrant/apps/ambra/README.md to test the ambra stack.

# Code Author Tasks:

> This list should be finished before code review:
- [x] Ticket AC is updated with any decisions made in comments or side conversations.
- [x] Testing or PO Accept notes that will assist the tester, Product Owner or reviewer are set in the ticket.
- [x] All PRs are created and linked for work I have done against other projects as part of this ticket.
- [x] Intergration Tests have been addressed.  QA has been notified of the feature and a ticket has been created for that work, or integration tests have been updated. 
- [x] There are unit tests sufficient for both positive and negative test cases, test coverage has not decreased as part of this work. 
- [x] Any necessary documentation in confluence or READMEs is updated.

# Code Reviewer Tasks
- [x] I read through the JIRA ticket's AC before doing the rest of the review.
- [x] I ran the code (in the review environment or locally). I agree the running code fulfills the Acceptance Criteria as stated on the JIRA ticket. If not practical I have seen some evidence that the code does what it says (tests have passed).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -366,17 +366,30 @@
     </resources>
     <plugins>
       <plugin>
-        <groupId>org.apache.tomcat.maven</groupId>
-        <artifactId>tomcat7-maven-plugin</artifactId>
-        <version>2.2</version>
+        <groupId>org.codehaus.cargo</groupId>
```
Comment (egh) [inline pom.xml]: There is no maven-tomcat8-plugin. This seems to be the preferred solution. It means you need to use `tomcat cargo:run` to run from the command line in your local dev environment.

**Comment by egh** [comment]
@sgatchell Sorry, forgot to add - can you also check that `mvn cargo:run` starts a dev server for you?

**Comment by sgatchell** [comment]
@egh confirmed that brings up wombat on port 8080:
```
mvn cargo:run -Dcargo.servlet.port=8080 -Dwombat.configDir=/etc/ambra
```

**Comment by egh** [comment]
@sgatchell thanks!

**Pull request merged**